workers/unity/Packages/com.improbable.gdk.buildsystem/Configuration/SpatialOSBuildConfiguration.cs

```diff
@@ -6,138 +6,94 @@
 
 namespace Improbable.Gdk.BuildSystem.Configuration
 {
-    [CreateAssetMenu(fileName = "SpatialOS Build Configuration", menuName = CreateMenuPath)]
-    public class SpatialOSBuildConfiguration : SingletonScriptableObject<SpatialOSBuildConfiguration>
-    {
-        internal const string CreateMenuPath = "SpatialOS/Build Configuration";
-
+    [CreateAssetMenu(fileName = "SpatialOS Build Configuration", 
+        menuName = EditorConfig.ParentMenu + "/" + EditorConfig.BuildConfigurationMenu)]
```

Comment:
Why not have `EditorConfig.BuildConfigurationMenu` do this concat'ing?

Reply:
attribute arguments need to be constant expressions :(